docs/sdk_benchmarks.md

```diff
@@ -0,0 +1,35 @@
+# SDK Usage Benchmarks
+
+## Concept
+
+The way we procured the data for our benchmarks was by profiling several popular open source Android applications 
+([GoogleIO](https://github.com/google/iosched), [RedReader](https://github.com/QuantumBadger/RedReader), [Fenix](https://github.com/mozilla-mobile/fenix)). 
+Same use case was performed on each application with and without Datadog SDK enabled.
+After each profiling session the data was saved and we extracted the relevant information for the CPU, RAM, NETWORK and BATTERY usage.
+
+The profiling sessions were performed on a SM-G960F device running Android version 10.
+
+## Benchmarks
+
+#### GoogleIO
+
+| Datadog SDK enabled | Max RAM | Max CPU | Max kb/s sent | Max kb/s received | Max number of connections | ENERGY consumption at peak |
```

Comment:
For the Max RAM / Max CPU values, are those values general from the phones (so impacted by other apps) or are they only taking into account the tested app ?

Reply:
For the Energy consumption, what does `< LIGHT` mean ? Since there's no quantified value, we should add some explanation on what Light means (e.g. is it taken from a predefined tool metrics, or is it an estimation)?

Reply:
For the network stats, I'm not sure the speed is relevant (`kb/s` is network speed right), especially since we don't know what network you're on (wifi ? 5g ? 3g ?). Instead maybe it would be better to list the number and size of requests (both upload and download). Meaning you should clear the cache before running the apps in each test to ensure you have the same amount of client network requests. 
The goal is to show how much more network request the SDK adds.

Reply:
Regarding CPU and RAM they're only related to the app process. The Android Profiler separates those values from the other processes.

Regarding the battery levels those are the levels used by the Android Profiler (LIGHT, MODERATE, HIGH) . Will add this in the footer note.